api/graphql/schema.graphql

```diff
@@ -242,7 +242,7 @@ type ScreeningQuestion {
 interface HasRoleAssignments {
   id: UUID!
   roleAssignments: [RoleAssignment!]
-  teamIdForRoleAssignment: ID # Used to assign roles associated with this resource using the updateUserRoles mutation.
+  teamIdForRoleAssignment: ID @with(relation: "team") # Used to assign roles associated with this resource using the updateUserRoles mutation.
 }
```

Comment:
I don't think this should go on the interface itself, as it isn't applicable in all cases, nor was it suggested by Tristan

Reply:
For example, team on team below

Reply:
Ahh okay thanks, removed in [remove overeager loading of teams](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11404/commits/8311cd5e745eb94b0c763ab24f38c6ce596287f6)